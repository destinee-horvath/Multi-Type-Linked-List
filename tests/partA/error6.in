NEW 4
2.5
{}
squid
$
NEW 3
{
a_word }
words