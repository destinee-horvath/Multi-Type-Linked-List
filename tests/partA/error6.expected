INVALID COMMAND: NEW
INVALID COMMAND: NEW